src/components/orders-table/orders-date-range-picker.tsx

```diff
@@ -1,37 +1,37 @@
 import { DatePicker, DatePickerInput } from '@carbon/react';
+import { useAppContext } from '@openmrs/esm-framework';
+import dayjs from 'dayjs';
 import React from 'react';
-import styles from './orders-date-range-picker.scss';
 import { useTranslation } from 'react-i18next';
+import { DateFilterContext } from '../../types';
+import styles from './orders-date-range-picker.scss';
 
-export const OrdersDateRangePicker = ({
-  onChange,
-  currentValues,
-}: {
-  onChange: (dates: Array<Date>) => void;
-  currentValues: Array<Date>;
-}) => {
+export const OrdersDateRangePicker = () => {
   const currentDate = new Date();
+  const { dateRange, setDateRange } = useAppContext<DateFilterContext>('laboratory-date-filter') ?? {
+    dateRange: [dayjs().startOf('day').toDate(), new Date()],
+    setDateRange: () => {},
+  };
+
   const { t } = useTranslation();
+
+  const handleOrdersDateRangeChange = (dates: Array<Date>) => {
+    setDateRange(dates);
+  };
+
   return (
-    <DatePicker
-      datePickerType="range"
-      className={styles.dateRangePicker}
-      onClose={onChange}
-      maxDate={currentDate.toISOString()}
-      value={currentValues}
-    >
-      <DatePickerInput
-        id="date-picker-input-id-start"
-        placeholder="mm/dd/yyyy"
-        labelText={t('startDate', 'Start date')}
-        size="md"
-      />
-      <DatePickerInput
-        id="date-picker-input-id-finish"
-        placeholder="mm/dd/yyyy"
-        labelText={t('endDate', 'End date')}
-        size="md"
-      />
-    </DatePicker>
+    <div className={styles.datePickerWrapper}>
+      <p>{t('dateRange', 'Date range')}:</p>
+      <DatePicker
+        datePickerType="range"
+        className={styles.dateRangePicker}
+        onClose={handleOrdersDateRangeChange}
+        maxDate={currentDate.toISOString()}
+        value={dateRange}
+      >
+        <DatePickerInput id="date-picker-input-id-start" placeholder="mm/dd/yyyy" size="md" />
+        <DatePickerInput id="date-picker-input-id-finish" placeholder="mm/dd/yyyy" size="md" />
```

Comment:
I know this has really dragged with the back and forth. Let's change this date format and get this in
```suggestion
        <DatePickerInput id="date-picker-input-id-start" placeholder="dd/mm/yyyy" size="md" />
        <DatePickerInput id="date-picker-input-id-finish" placeholder="dd/mm/yyyy" size="md" />
```

Reply:
This is to match other parts of the app